src/main/blackbox/blackbox.h

```diff
@@ -53,7 +53,7 @@ typedef enum FlightLogEvent {
 typedef struct blackboxConfig_s {
     uint16_t p_ratio; // I-frame interval / P-frame interval
     uint8_t device;
-    uint8_t record_acc;
+    uint16_t fields;
```

Comment:
If you change the structure of the parameter group you must increment it's version. See https://github.com/betaflight/betaflight/blob/master/docs/development/ParameterGroups.md

Also `fields` is a poor name and really doesn't indicate what it does. Maybe "logging_field_mask" or something like that.

Also, make it a `uint32_t`. There's absolutely no reason to restrict to only 16 bits when at some point later we'd only have to expand.

Reply:
Yep, sorry about the version, i always forget something

Regarding the 32bits, i left it at 16 as it has 4 unused bits and was thinking that these can be enough for some time. Maybe if i made it 32 bits i can do other fields selectable? Or it isn't necessary?

Reply:
There's no reason to not use 32 bits and plan for the future. Saving 2 bytes now is not worth the effort later to retrofit.